src/components/Cards/Cards.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+import { Link } from "react-router-dom";
+
+const Cards: FC = () => {
+  return (
+    <div>
+      <Link to="/product/1234">Cards</Link>
```

Comment:
all these "/product/1234" you can use constants

Reply:
Added